### PR TITLE
Connect MongoDB

### PR DESCRIPTION
**Pull request by jynlin3**
Test steps:
1. Checkout code.
2. Update dependencies for client: `npm install`
3. Install dependencies for backend: `npm run backend-install`
4. Put **config_backend.json** under **backend** folder. (**config_backend.json** is on Google Cloud or Facebook Messenger)
4. Run the client & backend with concurrently: `npm run dev`
5. Enter http://localhost:3000/word

### CODE REVIEW

File: backend/model.js [hunk 1]
```diff
@@ -9,4 +9,4 @@ const noun = new Schema({
     total_count: Number
 });
 
-module.exports = mongoose.model("noun", noun, "noun");
\ No newline at end of file
+module.exports = mongoose.model("noun", noun, "noun");
```
Comment (YaoChungLiang) [inline backend/model.js:12]: what is this

File: src/components/Word.js [hunk 2]
```diff
@@ -39,4 +39,4 @@ export default class Word extends Component{
     render(){
     return (<div><p>Generated word: {this.state.word}</p><p>Udpate result: {this.state.update_result}</p></div>);
     }
-}
\ No newline at end of file
+}
```
Comment (YaoChungLiang) [inline src/components/Word.js:42]: purpose of adding a new line?

File: backend/model.js (hunk 1 above)
Comment (jynlin3) [reply backend/model.js:12]: The call to `mongoose.model` ties to the same name of the collection in MongoDB. To use `noun` document in MongoDB, the call should look like the above.

File: src/components/Word.js (hunk 2 above)
Comment (jynlin3) [reply src/components/Word.js:42]: Sometimes the compiler may fail due to the lack of a newline at the end of the file, especially on linux.

**Pull request merged**